js/views/member_list_view.js

```diff
@@ -0,0 +1,64 @@
+/* global _, Whisper, */
+
+// eslint-disable-next-line func-names
+(function() {
+  'use strict';
+
+  window.Whisper = window.Whisper || {};
+
+  Whisper.MemberListView = Whisper.View.extend({
+    initialize(options) {
+      this.member_list = [];
+      this.selected_idx = 0;
+      this.onClicked = options.onClicked;
+      this.render();
+    },
+    render() {
+      if (this.memberView) {
+        this.memberView.remove();
+        this.memberView = null;
+      }
+
+      this.memberView = new Whisper.ReactWrapperView({
+        className: 'member-list',
+        Component: window.Signal.Components.MemberList,
+        props: {
+          members: this.member_list,
+          selected: this.selectedMember(),
+          onMemberClicked: this.handleMemberClicked.bind(this),
+        },
+      });
+
+      this.$el.append(this.memberView.el);
+      return this;
+    },
+    handleMemberClicked(member) {
+      this.onClicked(member);
+    },
+    update_members(members) {
```

Comment:
Surprised the linter didn't pick on that. Updated.

Reply:
I don't think we enabled that rule because signal has so many places where it uses `snake_case` and `camelCase` at the same time. Like object properties are `snake_case` and sometimes `camelCase`